### DISCUSSION TITLE
Preventing Filters from excluding date_shift data

### DISCUSSION THREAD

**Discussion by ncosgrov58** [atoti/atoti#762] [Q&A] [answered]
I have the following measure

```py
m["Price 30 Days Ago"] = tt.date_shift( m["Price.SUM"], h["Date"], offset="P1M", method="exact")
```

The problem I am having is that if  I go to a widget and try to show the price change just relative to today, i.e. Date-Today, I am then filtering out the 30 day old data the shift is using.    Is there a way to have this measure disregard the filter criteria?

**Comment by tibdex** [comment, marked as answer]
Hi, I tried to reproduce what I understood from what you said but I don't see your issue: Using your `Price 30 Days Ago` measure and filtering out some of these 30 past days in the UI doesn't change the value of the measure on the dates that are still there.

Can you share a minimal reproducible example and a screenshot of what you're getting and what you expect please?

**Comment by ncosgrov58** [comment]
Actually, your response helped me out I see my error and thank you.
